db/scripts/run

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+docker run --rm -d --network=host --name modelserver_db mrcide/modelserver_db:mrc-371
```

Comment:
This should be replaced by master, though the image is not currently being automatically built..